CefSharp.Example/ScriptedMethodsBoundObject.cs

```diff
@@ -35,5 +36,26 @@ public void RaiseEvent(string eventName, object eventData = null)
                 EventArrived(eventName, eventData);
             }
         }
+
+        /// <summary>
+        /// Demonstrates the use of params as an argument in a bound object
+        /// </summary>
+        /// <param name="name">Dummy Argument</param>
+        /// <param name="args">Params Argument</param>
+        public void MethodWithParams(string name, params object[] args)
```

Comment:
These should return a result, then output that result when they're called from `BindingTest.html`. Something similar to https://github.com/cefsharp/CefSharp/blob/cefsharp/47/CefSharp.Example/Resources/BindingTest.html#L210 would be ideal.